Essentials/src/com/earth2me/essentials/User.java

```diff
@@ -792,6 +792,7 @@ public void setVanished(final boolean set) {
                 }
             }
             setHidden(true);
+            this.getBase().setMetadata("vanished", new FixedMetadataValue(Essentials, true));
```

Comment:
"Essentials" is not a variable defined in this method. You'll need to get the instance to use it here.

Reply:
FixedMetadataValue is also not imported

Reply:
Hey okay will try, that is why I just wanted to make the feature request because I dont really know how to compile it from here. But will investigate. Sorry for disturbing and thanks!

Reply:
See: [Building](https://github.com/EssentialsX/Essentials#building)

Reply:
I just added the imports and put the plugin the way I saw they were doing in another classes (EssentialsPlayerListener for example). Sorry for being really noob about this :/
Now Ill try to see if I can compile it with maven, Im reading some docs to see if I can understand it